insights/parsers/uname.py

```diff
@@ -97,7 +97,9 @@
     "3.10.0-514": "7.3",
     "3.10.0-693": "7.4",
     "3.10.0-862": "7.5",
-    "3.10.0-957": "7.6"
+    "3.10.0-957": "7.6",
+    "4.18.0-70": "8.0",
+    "4.18.0-74": "8.0"
```

Comment:
These versions are only for RC# of RHEL-8, which won't be released. Only the version for GA is the target of insights.

Please use the RedHatRelease as a workaround. and close this PR.

Reply:
@xiangce - ack